snap/snapcraft.yaml

```diff
@@ -56,6 +56,9 @@ apps:
       TMPDIR: $XDG_RUNTIME_DIR
       # Fallback to XWayland if running in a Wayland session.
       DISABLE_WAYLAND: 1
+      # Force using the libsecret local backend in all the cases, even if no
+      # portal is detected.
+      SECRET_BACKEND: file
```

Comment:
Hmm we may want to ship this separately since it'll require everyone to re-authenticate their accounts again (since we won't read their previously saved passwords from the password manager service)

Is there a way we can verify libsecret is receiving an encryption master key via a portal? I mostly want to be be sure that Chrome doesn't fall through to https://chromium.googlesource.com/chromium/src/+/53.0.2785.100/components/os_crypt/os_crypt_linux.cc#71 and use the password `peanuts` for everyone!